hrui/hrui.php

```diff
@@ -934,3 +936,113 @@ function _hrui_menuSetActive($isActive) {
   CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET is_active = {$isActive} WHERE name = 'import_custom_fields'");
   CRM_Core_BAO_Navigation::resetNavigation();
 }
+
+function _getMenuKeyMax($menuArray) {
+  $max = array(max(array_keys($menuArray)));
+  foreach($menuArray as $v) {
+    if (!empty($v['child'])) {
+      $max[] = _getMenuKeyMax($v['child']);
+    }
+  }
+  return max($max);
+}
+
+function civicrm_civicrm_navigationMenu(&$menu) {
+  // Create New Help Menu
+  _createHelpMenu($menu);
```

Comment:
Comments are good if they add some extra information, but I think the function names are clear enough here.